e2e/playwright/support/ui/pages/boards_view.ts

```diff
@@ -53,6 +60,32 @@ export default class BoardsViewPage {
         expect(await this.editableTitle.getAttribute('value')).toBe('');
         await expect(this.page.getByTitle('(Untitled Board)')).toBeVisible();
     }
+
+    async createDuplicateBoard(boardName: string) {
+        await this.page.hover(`div[title='${boardName}']`);
+        await this.page.locator(`//div[@title='${boardName}']/following-sibling::div`).click();
```

Comment:
These were repeated in 2 places. I'd suggest to have common access method in Boards [sidebar](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/playwright/support/ui/components/boards/sidebar.ts) component, like:
```javascript
async getBoardItem(name: string) {
    return await this.container.locator('.SidebarBoardItem').filter({hasText: name});
}
```

The these can be
```suggestion
        // Hover over board item
        const sidebarBoard = await this.sidebar.getBoardItem(boardName);
        await expect(sidebarBoard).toBeVisible();
        await sidebarBoard.hover();

        // Click its option button to open menu
        await sidebarBoard.locator('.IconButton').click();

        // Click "Duplicate board"
```